lib/charty/color_methods.rb

```diff
@@ -0,0 +1,85 @@
+module Charty
+  module ColorMethods
+    def RGB(*args)
+      case args.length
+      when 1
+        begin
+          return args[0].to_rgb
```

Comment:
How about removing `return` and use `else` for `raise ArgumentError, "wrong number ..."`?